test/examples/test_examples.py

```diff
@@ -35,6 +35,7 @@
     'lockExchange/plotting.py',
     'lockExchange/submitRuns.py',
     'migrating_trench/trench_hydro.py',
+    'meander/meander_hydro.py',
```

Comment:
why are the hydro examples omitted from tests? the scripts do have the test switch

Reply:
The hydro files are run via the main scripts e.g. line 65 in migrating_trench/trench_example.py

Reply:
ok, I see